ui/src/components/Filters.vue

```diff
@@ -1027,8 +1027,6 @@ export default {
         // set agents filter based on selected queues
         this.filter.queues.length > 0 ? (
           this.filter.agents = this.agentsFilteredByQueue.map(agent => agent.value)
-        ) : (
-          this.filter.agents = []
```

Comment:
The syntax is wrong use a single if in this case https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else